addon/ng2/tasks/npm-install.js

```diff
@@ -0,0 +1,17 @@
+/* jshint node: true, esversion: 6 */
```

Comment:
Stefan answered that issue with a request to follow their template.

Also, where did you find out about that bug? Did you test it yourself? Could you try with ember-cli (without angular) to make sure it's not a bug on our side?